crates/vm/levm/src/vm.rs

```diff
@@ -558,18 +562,19 @@ impl VM {
                 .ok_or(OutOfGasError::ConsumedGasOverflow)?;
 
             let number_of_words = initial_call_frame.calldata.len().div_ceil(WORD_SIZE);
+            let double_nuber_of_words: u64 = number_of_words
```

Comment:
```suggestion
            let double_number_of_words: u64 = number_of_words
```
Suggestion, in this kind of PRs that are very big maybe it is best not to do these kind of changes, but I agree that it looks better to have this in another variable.

Reply:
Noted Jere, I only added a new variable in the cases where a variable was overloaded with method calls after I added the new types. 

Should I keep _this_ specific variable? I'll change the rest

Reply:
Spellcheck: https://github.com/lambdaclass/ethrex/pull/1528/commits/bab45f27990b7a9ef3c86cb6c5b33da442ec4dcc